src/scripts/main.js

```diff
@@ -358,3 +358,43 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the HTML document contains an element with the class 'dashboard'. If not, the `querySelector` method will return `null`, and attempting to append children to `null` will result in an error. Consider adding a check to ensure `table` is not `null` before proceeding.